apps/dapp/src/hooks/core/use-withdraw-from-vault.tsx

```diff
@@ -14,21 +14,21 @@ export const useWithdrawFromVault = (vaultContractAddress: string, onSuccess?: C
   const { openNotification } = useNotification();
   const { optimisticallyUpdateVaultStaked } = useVaultContext();
 
-  const withdrawEarly = async (subvaultAddress: string, amount: BigNumber) => {
+  const withdrawEarly = async (subvaultAddress: string, amount: string) => {
     if (!signer || !wallet) {
       console.error(`
         Attempted to withdraw early from vault: ${env.contracts.vaultEarlyExit} without a valid signer.
       `);
       return;
     }
 
+    const bigAmount = getBigNumberFromString(amount);
     const vaultEarlyWithdrawal = new VaultEarlyWithdraw__factory(signer).attach(env.contracts.vaultEarlyExit);
 
-    const receipt = await vaultEarlyWithdrawal.withdraw(subvaultAddress, amount, { gasLimit: 400000 });
-
+    const receipt = await vaultEarlyWithdrawal.withdraw(subvaultAddress, bigAmount, { gasLimit: 400000 });
```

Comment:
Out of interest - does ethers get the gas estimate wrong here, is that why you need to specify `gasLimit`?

Reply:
Basically yes, that's what was happening when I was manually testing.